constraint_utils/constraint_basics.py

```diff
@@ -227,6 +247,15 @@ def filter_to_autosomes_par(ht: Union[hl.Table, hl.MatrixTable]) -> Union[hl.Tab
 
 def annotate_with_mu(ht: hl.Table, mutation_ht: hl.Table, output_loc: str = 'mu_snp',
                      keys: Tuple[str] = ('context', 'ref', 'alt', 'methylation_level')) -> hl.Table:
+    """Annotate SNP mutation rate for the input Table.
```

Comment:
I think KC pointed this out on your other PR, in general if you learn something from another PR and you see that it's also a problem in another PR you have, it is good to modify it as well.

```suggestion
    """
    Annotate SNP mutation rate for the input Table.
```